driver-core/src/main/java/com/datastax/driver/core/ExecutionInfo.java

```diff
@@ -87,7 +88,10 @@ ExecutionInfo withIncomingPayload(Map<String, ByteBuffer> incomingPayload) {
      * {@link com.datastax.driver.core.policies.RetryPolicy} may retry the
      * query on the same host, so the same host might appear twice.</li>
```

Comment:
> RequestTimeline

That would be **great** it's kind of what I was imagining when opening the ticket